R/calculate_logit_inconv_endog.R

```diff
@@ -385,6 +386,10 @@ calculate_logit_inconv_endog = function(prices,
     ## overwrite the preferences with the market based ones
     pref_data[["FV_final_pref"]] = final_prefFV
 
+    ##HOTfix: Truck size classes are not all included in the input data -> they need to be added manually
+    ##ATTENTION: This should be changed in the refactoring process
+    dfother[grepl("Truck", vehicle_type), logit.exponent := -4]
```

Comment:
```suggestion
    dfother[grepl("^Truck", vehicle_type), logit.exponent := -4]
```

Reply:
In this case I agree with Marianna that explicitly including only vehicle_types that start with `Trucks` is less ambiguous although technically you are right that they are not affected.